01/assets/js/app.js

```diff
@@ -1,4 +1,28 @@
-const a = '4.2';
+const a = Number("4.2");
```

Comment:
Warto też sprawdzić jak to wszystko będzie działać jak nie będzie konwersji tylko samo `const a = '4.2'`